projects/js-packages/connection/components/connect-user/index.jsx

```diff
@@ -63,41 +48,24 @@ const ConnectUser = props => {
 		return null;
 	}
 
-	if ( forceCalypsoFlow ) {
-		redirectFunc(
-			authorizationUrl +
-				( from
-					? ( authorizationUrl.includes( '?' ) ? '&' : '?' ) + 'from=' + encodeURIComponent( from )
-					: '' )
-		);
-		return null;
-	}
-
-	return (
-		<InPlaceConnection
-			connectUrl={ authorizationUrl }
-			title={ inPlaceTitle }
-			onComplete={ onComplete }
-			displayTOS={ displayTOS }
-		/>
+	redirectFunc(
```

Comment:
@sergeymitr Mostly out of curiosity, does it still make sense to have `redirectFunc`? Trying to think of a possible case where it would make sense to provide anything else than the default `window.location.assign` 🤔 
Should we maybe enforce the redirect by removing this property?

Reply:
That's needed for testing.
I know it's not the best practice, but I wasn't able to properly mock it, so I extracted the redirect into a parameter with `window.location.assign()` being its default value.